README.md

```diff
@@ -243,6 +243,7 @@ Availability:
 
 Availability:
 - `--py3-plus` is passed on the commandline.
+- Can be suppressed with `--no-mock`. See [#314](https://github.com/asottile/pyupgrade/issues/314) for more context.
```

Comment:
I'd just say `- [Unless `--no-mock` is passed on the commandline`]()`

Reply:
Sure, should I provide more context on why we have this flag? Either by linking to the PR or summarizing something like "`AsyncMock` shouldn't be rewritten in certain versions and for consistent behavior across files you can disable rewriting mock entirely"

Reply:
oh, my template was supposed to look like a link but I forgot to paste the link into it -- I'd just link to the issue